src/gp/inc/GPMSA.h

```diff
@@ -364,13 +364,16 @@ class GPMSAFactory
   // All the GP priors information for a scalar GP follows:
   void setUpHyperpriors();
 
+  // Number of dimensions preserved from the SVD of simulation outputs
+  unsigned int num_svd_terms;
+
   // Domains for all the hyperpriors
   typename ScopedPtr<VectorSpace<V, M> >::Type oneDSpace;
 
-  // Emulator mean
-  typename ScopedPtr<V>::Type emulatorMeanMin;
-  typename ScopedPtr<V>::Type emulatorMeanMax;
-  typename ScopedPtr<BoxSubset<V, M> >::Type emulatorMeanDomain;
+  // Truncation error precision
+  typename ScopedPtr<V>::Type truncationErrorPrecisionMin;
+  typename ScopedPtr<V>::Type truncationErrorPrecisionMax;
+  typename ScopedPtr<BoxSubset<V, M> >::Type truncationErrorPrecisionDomain;
```

Comment:
If the emulator mean is being removed altogether, then what is `truncationError*` for?